espressif/vscode-esp-idf-extension
# add gitee links

## Description

Add gitee mirrors for ESP-IDF download for Espressif download server in extension setup.

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

## Steps to test this pull request

Provide a list of steps to test changes in this PR and required output

1. Click on "ESP-IDF: Configure ESP-IDF extension"
2. Execute setup using the `Espressif` download server.
3. Observe results. ESP-IDF should be clone from gitee for master and release branches (which use git clone download strategy)

- Expected behaviour: Cloning of ESP-IDF from gitee servers.

## How has this been tested?

Manual test of ESP-IDF master branch install

**Test Configuration**:
* ESP-IDF Version: 5.0
* OS (Windows,Linux and macOS): macOS

## Checklist
